packages/workflow/src/interfaces.ts

```diff
@@ -140,7 +140,7 @@ export interface UnsafeWorkflowInfo {
    * Current system time in milliseconds
    *
    * The safe version of time is `new Date()` and `Date.now()`, which are set on the first invocation of a Workflow
-   * Task.
+   * Task and stay constant.
```

Comment:
What do you mean stay constant? This is inaccurate.

Reply:
per-Task constant/consistent, vs system/real.

is this clearer?

`stay constant for the duration of the Task and during replay. `

Reply:
Yes